### ISSUE TITLE
Music Bot

### ISSUE THREAD

**Issue by dustinblackman** [Gravestorm/Gravebot#118]
DiscordJS makes is very easy to queue up a music bot using it's [playFile](https://discordjs.readthedocs.org/en/latest/docs_voiceconnection.html#playfile-path-options-callback) feature.

**Comment by ryanwcraig** [comment]
I, for one, would _love_ to have a voice channel playing the song Paul Ryan put his blizzard livestream to.

**Comment by dustinblackman** [comment]
Haha it's on our list, we just have a few other core features were trying to push out first.

**Comment by ryanwcraig** [comment]
Can non-collaborators work on this, or is that limited to those already on the project?

**Comment by dustinblackman** [comment]
Nope! Project is open source. Anyone is free to contribute. :) Check this out. https://github.com/Gravestorm/Gravebot/blob/master/CONTRIBUTE.md

**Comment by ryanwcraig** [comment]
Awesome!

**Comment by sh1omi** [comment]
Is there a news about this music feature?

**Comment by dustinblackman** [comment]
@sh1omi Bigger things to accomplish first, it's on the back burner. PRs are welcome.

**Comment by ryanwcraig** [comment]
I had experimented with this and got *something* working, but it kind of fell off my radar. I'll see what source I still have available.

**Comment by jtagt** [comment]
I feel like you could implement lavaplayer with lavalink to help with the load for music.
